src/TemplateRender.js

```diff
@@ -149,12 +149,19 @@ TemplateRender.prototype.render = async function(str, data) {
   return this.engine.render(str, data);
 };
 
-TemplateRender.prototype.getCompiledTemplate = async function(str) {
+TemplateRender.prototype.getCompiledTemplate = async function(str, inputPath) {
   // TODO refactor better, move into TemplateEngine logic
   if (this.engineName === "md") {
-    return this.engine.compile(str, this.parseMarkdownWith, !this.useMarkdown);
+    return this.engine.compile(
```

Comment:
You’re calling `compile` assuming an incorrect signature. The signature is:

```js
compile(str, inputPath, preTemplateEngine, bypassMarkdown)
```

But you pass `inputPath` last.

Reply:
Yup, that was it!